stable/openstack-base/openstack-base-monitoring-pgt-overlay.yaml

```diff
@@ -0,0 +1 @@
+../overlays/openstack-base-monitoring-pgt-overlay.yaml
```

Comment:
I don't think this overlay is core/fundamental enough to deserve a symlink here. Other IMHO more important overlays like Octavia don't have a symlink here.

Apart from that I think the whole patch now looks good, thanks :)